### DISCUSSION TITLE
Can't type() into contenteditable element.

### DISCUSSION THREAD

**Discussion by nminev** [cypress-io/cypress#26525] [Questions and Help]
**Context:** I'm using cypress 10.11.0 and am trying to select an element on ck editor 5. 

I have a div that has the contenteditable attributes and am trying to type into it, but at run time the type fails and the error suggest that the div doesn't have the attribute, but even right before the type() call it still had it. Doesn't seem to be a problem with CK Editor since it's just html at this point. 

Code:
```
const body =
        'div.ck-editor__editable[contenteditable="true"]';
cy.get(body)
.should('be.visible')
.click()
.focused()
.wait(10000) // Wait for 10 second to check attribute
.type('This is additional text', { force: true })
.type('{ctrl+a}');

```
Screenshot of the error:
![screenshot of contenteditable](https://user-images.githubusercontent.com/17933687/232728211-fa9bb69e-a7d9-46d4-9dd8-591f2583c2af.png)

PS : I've seen [this ](https://github.com/cypress-io/cypress/discussions/26021) issue but I don't think it's quite the same

**Comment by WalterWoshid** [comment]
Bump
